### ISSUE TITLE
Email Op-ins 404

### ISSUE THREAD

**Issue by mstratford** [UniversityRadioYork/MyRadio#674]
Trying to goto https://ury.org.uk/myradio/mail/optin/?list=43
(clicking + on main manager)

**Comment by mstratford** [comment]
Possibly should possibly be fixed here: #668?

**Comment by LordAro** [comment]
```
[01/Apr/2017:11:36:25 +0100] "GET /myradio/mail/optin/?list=43 HTTP/2.0" 404 1170 "https://github.com/UniversityRadioYork/MyRadio/issues/674" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36" "-"
[01/Apr/2017:11:36:25 +0100] "GET /myradio/MyRadio/config.js HTTP/2.0" 200 786 "https://ury.org.uk/myradio/mail/optin/?list=43" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36" "-"
```
